README.md

```diff
@@ -0,0 +1,55 @@
+# SciData
```

Comment:
Sorry for the nitpick. If we want to call it `SciData` then the convention is to call the application (and the repository) `sci_data` - with underscore. We can easily adjust it though. Which one do you prefer? :)

Reply:
I think `Scidata` is good! I'll change the module names.